xrefs.tsv

```diff
@@ -261,3 +261,6 @@ CONSO00360	ncit	C20613
 CONSO00361	ncit	C113880
 CONSO00361	umls	C3827230
 CONSO00362	antibodyregistry	AB_2314773
+CONSO00363	pdb	4OZ4
+CONSO00363	cat.no	PABL-712
```

Comment:
these aren't real namespaces from Identifiers.org. I'm not sure the best way to add this information. Why don't you use `url` as the namespace and actually add a link to the page